src/test/lib/statistics/attribute_statistics_test.cpp

```diff
@@ -117,8 +143,8 @@ TEST_F(AttributeStatisticsTest, OutputToStream) {
 Generic value count: 40; distinct count: 20; bin count: 1;  Bins
   [1 -> 100]: Height: 40; DistinctCount: 20
 
-Has MinMaxFilter
-Has RangeFilter
+MinMaxFilter: {0 100}
+RangeFilter: { 0->100 }
```

Comment:
Not a big fan of this input. As we should test more ranges IMO.

Reply:
Added another range